src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Event } from "./event/event";
 
 function App() {
   return (
-    <div className="App">
+    <div className="App min-vh-100 d-flex flex-column align-items-center">
       <header className="App-header">
```

Comment:
I know you didn't touch this part, but `App-header` appears to be a dead class and can be removed.

Reply:
Good eye on that, removed.